inc/functions_upload.php

```diff
@@ -722,7 +722,7 @@ function add_attachments($pid, $forumpermissions, $attachwhere, $action=false)
 		{
 			foreach($fields as $field)
 			{
-				$attach1[$field] = $_FILES['attachments'][$field][$key];
+				$attach1[$field] = $_FILES['attachments'][$field][$i];
```

Comment:
Looks like `$attach1` is never used in this function, maybe legacy code.

Reply:
Yeah, that's what I saw as well looking at it.  I see a few other undefined variables in other areas but probably have to someone look at them for the fix.

Reply:
If it has no use then it should be removed.

Reply:
Done removed the line